### PR TITLE
Add Mute Button to Multiplayer Game Page

### PR DESCRIPTION
**Pull request by thsparks**
This adds a mute button to the multiplayer game page. To make room for it, I also had to move the "Leave Game" button to the header bar and shift the join code to the center of the screen.

image

image

### CODE REVIEW

File: multiplayer/src/components/GamePage.tsx [hunk 1]
```diff
@@ -52,49 +58,50 @@ export default function Render(props: GamePageProps) {
...
-                                                className="tw-text-green-600 tw-mb-[0.1rem]"
-                                            />
-                                        )}
-                                    </div>
-                                </button>
-                            </div>
-                        )}
-                        <div className="tw-flex-grow" />
+                    <div className="tw-flex tw-flex-row tw-w-full tw-px-2 tw-items-center tw-justify-between">
+                        <button
```
Comment (eanders-ms) [inline multiplayer/src/components/GamePage.tsx:62]: Let's use react-common's Button here. It's set up with all the needed accessibility props. You might want to inject the font awesome icon as a JSX.Element to the `label` prop.

**Review by jwunderl — APPROVED** [review]

File: multiplayer/src/components/GamePage.tsx [hunk 2]
```diff
@@ -52,49 +59,52 @@ export default function Render(props: GamePageProps) {
...
+                            }
+                            title={lf("Toggle Mute")}
+                            className="tw-border-2 tw-border-slate-400 tw-border-solid tw-py-2 tw-pl-2 tw-pr-1 tw-bg-slate-100 hover:tw-bg-slate-200 active:tw-bg-slate-300"
+                            onClick={toggleMute}
+                        />
+                        <div className="tw-justify-self-center">
+                            {state.gameState?.joinCode && (
+                                <div>
+                                    {state.gameState?.joinCode &&
+                                        `${lf("Join Code")}: ${
```
Comment (eanders-ms) [inline multiplayer/src/components/GamePage.tsx:77]:
```suggestion
lf("Join Code: {0}", state.gameState?.joinCode)
```

**Review by eanders-ms — APPROVED** [review]
LGTM! Just one small suggestion.

**Comment by thsparks** [comment]
Follow-up item: https://github.com/microsoft/pxt-arcade/issues/5176

Not doing it in this PR because I think there are more important issues to address at the moment.

**Comment by thsparks** [comment]
Fixes https://github.com/microsoft/pxt-arcade/issues/5161

**Pull request merged**